src/cli.ts

```
@@ -1,13 +1,17 @@
 import semver from "semver";
+import { type InferInput, parseAsync } from "valibot";
 import yargs, { type Argv } from "yargs";
 import pkg from "../package.json" with { type: "json" };
+import { MojisNotImplemented } from "./adapter";
 import { resolveAdapter } from "./adapters";
 import { SUPPORTED_EMOJI_VERSIONS } from "./constants";
-import { getAllEmojiVersions } from "./utils";
-import { readLockfile, writeLockfile } from "./utils/lockfile";
+import { SHORTCODE_PROVIDERS_SCHEMA } from "./schemas";
+import { getAllEmojiVersions, getUnicodeVersionByEmojiVersion } from "./utils";
+import { readLockfile, writeLockfile } from "./lockfile";
```

ESLint

[error] Expected "./lockfile" to come before "./utils".

(perfectionist/sort-imports)